### ISSUE TITLE
Strange issue blocking Auth0

### ISSUE THREAD

**Issue by mikehuebner** [newrelic/newrelic-react-native-agent#32]
Hey all!

My team just started trying to utilize NewRelic for our React Native application and we learned that after we call `NewRelic.startAgent(appToken, agentConfiguration);` we cannot seem to login with Auth0. We get an `unknown error` from the service and a `response.ok` of `false`. So far we have only seen this behavior with Auth0 and wondering what we can do to fix this. We did not see any specific steps to get NewRelic working with Auth0 so if we missed something please let us know! 😄

**Comment by ndesai-newrelic** [comment]
@mikehuebner we have not seen any issue with Auth0, can you provide us more details?

**Comment by mikehuebner** [comment]
Just adding our code to showcase how this is working currently 😄 
```
const loginMutation = useMutation({
    mutationFn: (credentials: UserCredentials) =>
      auth0.auth.passwordRealm({
        ...credentials,
        realm: CONFIG.auth0.connection,
        scope: CONFIG.auth0.scope,
        audience: CONFIG.auth0.audience, // returns Opaque Access Tokens without this param
      }),
    onSuccess: (credentials) => {
      setUser(credentials);
    },
    onError: (err) => {
      console.warn('Error logging in:', err);
      // NewRelic.recordError(err);
    },
  });
```

**Comment by ndesai-newrelic** [comment]
@mikehuebner are you unable to login with our agent? what error are you getting?

**Comment by mikehuebner** [comment]
Here is the comparison

Agent on:
```
{
   "headers":{
      "map":{
         
      }
   },
   "json":{
      "access_token":"...",
      "expires_in":1111111,
      "id_token":"...",
      "refresh_token":"...",
      "scope":"openid profile email offline_access",
      "token_type":"Bearer"
   },
   "ok":false,
   "status":0
}
```

Agent off:  
```
{
  "headers":{
     "map":{
        "alt-svc":"h3=\":443\"; ma=86400, h3-29=\":443\"; ma=86400",
        "cache-control":"no-store",
        "cf-cache-status":"DYNAMIC",
        "cf-ray":"766941cd5c59e997-DFW",
        "content-encoding":"br",
        "content-type":"application/json",
        "date":"Mon, 07 Nov 2022 21:45:38 GMT",
        "ot-baggage-auth0-request-id":"766941cd5c59e997",
        "ot-tracer-sampled":"true",
        "ot-tracer-spanid":"0cf124d30bb0907b",
        "ot-tracer-traceid":"5e9562f93a805cf2",
        "pragma":"no-cache",
        "server":"cloudflare",
        "strict-transport-security":"max-age=31536000",
        "traceparent":"00-00000000000000005e9562f93a805cf2-0cf124d30bb0907b-01",
        "tracestate":"auth0-request-id=766941cd5c59e997,auth0=true",
        "vary":"Accept-Encoding, Origin",
        "x-auth0-requestid":"6d129d3c3978709093bc",
        "x-content-type-options":"nosniff",
        "x-ratelimit-limit":"100",
        "x-ratelimit-remaining":"99",
        "x-ratelimit-reset":"1667858403"
     }
  },
  "json":{
     "access_token":"...",
     "expires_in": 1111111111,
     "id_token":"...",
     "refresh_token":"...",
     "scope":"openid profile email offline_access",
     "token_type":"Bearer"
  },
  "ok":true,
  "status":200
}
```

I think I'm onto something with the headers.

**Comment by ndesai-newrelic** [comment]
@mikehuebner can you try those request on postman without header and share results with us?

**Comment by nandhakumar-thiyagarajah** [comment]
How to off agent..can you tell me ? i am facing the same issue as well.

**Comment by kthai-nr** [comment]
@nandhakumar-thiyagarajah Unfortunately, we don't currently can't stop the agent once it's started but we are looking to add that feature in the future. Are you and @mikehuebner getting this issue only on iOS or android as well? 

We've been seeing some issues pop up from Flipper causing issues in network instrumentation on iOS (#44) . I've gotten Auth0 to work on iOS when disabling Flipper in `Podfile` by commenting out this line:

```
    # Note that if you have use_frameworks! enabled, Flipper will not work and
    # you should disable the next line.

    :flipper_configuration => FlipperConfiguration.enabled,
```

**Comment by kthai-nr** [comment]
Hello, @nandhakumar-thiyagarajah @mikehuebner this issue should be fixed in release v1.1.0 without disabling Flipper. You can find the release here: https://www.npmjs.com/package/newrelic-react-native-agent/v/1.1.0. Please let us know if you run into any more issues.
